tests/cupyx_tests/scipy_tests/sparse_tests/test_index.py

```diff
@@ -123,6 +130,22 @@ def test_major_all_minor_slice(self):
     def test_major_all_minor_all(self):
         self._run(slice(None), slice(None))
 
+    # Major Indexing
```

Comment:
Could you add more tests for checks of `_unpack_index`? `IndexMixin.__getitem__` allows lists of bool/int, boolean mask of {numpy,cupy}.ndarray and integer {numpy/cupy}.ndarray.

Reply:
I overlooked during the review of the previous PR, but we want to test also for NumPy's scalar input, not only Python's primitive int.

Reply:
I just realized I missed this one in the tests. I also need to handle a scipy.sparse import and make it conditional.